core/src/main/java/com/redhat/lightblue/client/response/LightBlueResponseErrorCodes.java

```diff
@@ -0,0 +1,191 @@
+package com.redhat.lightblue.client.response;
+
+public class LightBlueResponseErrorCodes {
```

Comment:
Pls make this an interface, and have the exception class implement it for now. There are clients using the constants in the exception class.